<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="SUBHASHA-PC" failures="0" tests="5" name="com.nt.examples.TestNgExample1" time="0.024" errors="1" timestamp="2022-07-16T09:36:45 IST" skipped="0">
  <testcase classname="com.nt.examples.TestNgExample1" name="logOut" time="0.001">
    <error message="Cannot invoke &quot;org.openqa.selenium.WebDriver.close()&quot; because &quot;com.nt.hrm.TestBase.driver&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebDriver.close()" because "com.nt.hrm.TestBase.driver" is null
at com.nt.hrm.TestBase.logOut(TestBase.java:31)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:344)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- logOut -->
  <system-out/>
  <testcase classname="com.nt.examples.TestNgExample1" name="UpdateUser" time="0.004"/>
  <system-out/>
  <testcase classname="com.nt.examples.TestNgExample1" name="AddUser" time="0.007"/>
  <system-out/>
  <testcase classname="com.nt.examples.TestNgExample1" name="searchUser" time="0.001"/>
  <system-out/>
  <testcase classname="com.nt.examples.TestNgExample1" name="deleteUser" time="0.011"/>
  <system-out/>
</testsuite> <!-- com.nt.examples.TestNgExample1 -->
